internal/workload/wrapper.go

```diff
@@ -129,9 +140,65 @@ func (ww Workload) Run(workload *v1.Pod, manifestPath string, authFilePath strin
 	if err != nil {
 		return err
 	}
+
+	// Create the system service to manage the pod:
+	svc, err := ww.addService(workload)
+	if err != nil {
+		return err
+	}
+	err = ww.serviceManager.Add(svc)
+	if err != nil {
+		return err
+	}
+
 	return ww.mappingRepository.Add(workload.Name, podIds[0])
 }
 
+func (ww Workload) removeService(workloadName string) error {
+	svc := ww.serviceManager.Get(workloadName + "_pod")
```

Comment:
`workloadName + "_pod"` repeats in the code multiple times; WDYT about putting it in a function?